doc/source/migrations/migrate0_1.rst

```diff
@@ -21,24 +21,24 @@ and can be advantageous.
 See :issue:`504` for more details.
 Due to this it was decided that the best way forward was to abandon the old design,
 and to create a brand new data structure.
-This means that backwards compatibility *will* be broken, 
-and so this fix is leading to a major version release.
+This means that backwards compatibility *was* broken, 
+and so this fix lead to a major version release.
```

Comment:
leads

Reply:
I feel like this should be past tense. This is being written for 1.0.0 release so 1.0.0 will be in the past for this.

Reply:
~~leads~~ led

Reply:
I'm good at english.